gammacat/input.py

```diff
@@ -216,6 +216,47 @@ def validate(self):
         log.info('Validating YAML files in `input/sources`')
         [_.validate() for _ in self.data]
 
+class InputInfo:
```

Comment:
This class handles one of the info files which are present in the input folder.
Basically, the only thing that it is being possible to do is to validate the file (but in the future there may be more functionality).